src/components/Header/Header.css

```diff
@@ -5,13 +5,14 @@
   display: flex;
   justify-content: center;
   text-transform: uppercase;
-  background-color: grey;
+  background-color: #282c34;
   font-family: "Montserrat", sans-serif;
-  font-size: 6vw;
-  padding-bottom: 15px;
-  box-shadow: 0px 1px 5px black;
+  font-size: 3vw;
+  font-weight: 900;
+  box-shadow: 0px 1px 5px white;
   color: white;
   z-index: 100;
```

Comment:
I'm not sure what you want to achieve, but z-index only work with **positioned** elements. And this one is not positioned. So it's basically not doing anything right now

Reply:
I deleted it.

Reply:
Without Z-Index 100 the content of the main part goes over the header.